### DISCUSSION TITLE
Mailchimp for WooCommerce Opt-in Newsletter Checkbox Missing

### DISCUSSION THREAD

**Discussion by jfidelsolutions** [woocommerce/woocommerce#45092] [WooCommerce, Blocks, and Site Editing]
Hello, 

After running some tests I've realized that when I have WooCommerce Checkout Block active the (Mailchimp for WooCommerce plugin) Opt-In newsletter checkbox is missing from my checkout page. When I deactivated the plugin the checkbox automatically reappeared. Please help me troubleshoot this! Thank you.

**Comment by senadir** [comment]
Hi there @jfidelsolutions  :wave:

Thanks for taking the time to make us aware that Mailchimp for WooCommerce isn’t working with the Checkout and Cart blocks. These blocks are still in early access (via the feature plugin) and it’s expected that some existing Woo Extensions that integrate with the current checkout flow in WooCommerce core will not work with the blocks.

We’re currently working on adding extensibility for the blocks so extensions have a way to make sure they integrate with the new flows we’re developing. Once that is done we’ll have docs prepared and will be reaching out to Woo Extension developers to help them with getting their extensions integrated.

Mailchimp is on our radar and we're going to work on it soon.

Until then you can keep on top of the [status of the Cart & Checkout Blocks in this documentation](https://docs.woocommerce.com/document/cart-checkout-blocks-support-status/)

**Comment by nielslange** [comment]
Similar request via https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/6284.

**Comment by ralucaStan** [comment]
Hey @jfidelsolutions,

The MailChimp for WooCommerce is now compatible with the Cart and Checkout Blocks.

The minimum requirements are:

WooCommerce Blocks 8.0.0+
Mailchimp for WooCommerce 2.7+

You can read and find other compatible extensions with Cart & Checkout blocks here: https://woocommerce.com/document/cart-checkout-blocks-support-status/#section-5

Should you have any questions, feel free to reach out to us and we will be happy to assist you! 🙂
